tests/test_sf_sql.py

```diff
@@ -0,0 +1,1323 @@
+#
```

Comment:
Not the testing suite won't work in this setup. See comment about **https://docs.snowflake.com/en/developer-guide/snowpark/python/testing-locally**